.circleci/config.yml

```diff
@@ -328,7 +325,7 @@ jobs:
           keys:
             - cache-key-benchmarks-{{ checksum "setup.py"}}-{{ checksum "requirements-test.txt"}}
 
-      - <<: *install_dep_17_gpu
+      - <<: *install_dep_17
```

Comment:
Great catch! I think we need a cmd to check for the expected version and if mismatch failing it. I can add it in my PR if you don't have time adding it here.

Reply:
I think that some ideas from @anj-s would be a natural match, if we collect the results of the runs somewhere then catching the non-runs would be kind of easy, we would just see (or assert) that there are no results really. Right now the CI is "fire and forget", as long as the job does not crash it's green, even if it actually did nothing :/